lib/Mailer.php

```diff
@@ -108,7 +107,7 @@ public function send(MessageInterface $message)
         $response = $this->doSend($message);
 
         // We are all clear if status is HTTP 2xx OK
-        if ($response->isSuccessful()) {
+        if ((bool) in_array($response->getStatusCode(), range(200, 206), true)) {
```

Comment:
Isn't `in_array` always returning a `bool`?

Reply:
It is.. I just copied code from Response without caring too much. 
Thank you